docs/contributing.md

```diff
@@ -0,0 +1,71 @@
+# Contributing definitions
+
+Contributing library definitions is as easy as sending a pull request!
+
+All definitions sit under the
+[/definitions](https://github.com/flowtype/flow-typed/tree/master/definitions)
+directory. They all must follow the following directory structure and naming
+format:
+
+```
+└ definitions/npm/
+  ├ yargs_v4.x.x/           # <-- The name of the library, followed by _v<VERSION>
+  | |
+  | ├ flow_v0.23.x/         # <-- A folder containing libdefs tested against the
+  | | |                     #     specified version(s) of Flow (v0.23.x in this
+  | | |                     #     case).
+  | | |
+  | | └ yargs_v4.x.x.js     # <-- The libdef file meant for the Flow version
+  | |                       #     specified by the containing directory's name.
+  | |                       #     Must be named `<LIB>_v<VERSION>.js`.
+  | |
+  | ├ flow_v0.19.x-v0.22.x/ # <-- A folder containing libdefs tested against a
+  | | |                     #     different range of Flow versions:
+  | | |                     #     Anything from v0.19.x to v0.22.x (inclusive)
+  | | |
+  | | ├ yargs_v4.x.x.js     # <-- The libdef file for versions of Flow from
+  | | |                     #     v0.19.x to v0.22.x (inclusive)
+  | | |
+  | | └ test_yargs.js       # <-- Tests in this directory only apply to the
+  | |                       #     adjacent libdef (and thus, are specific to
+  | |                       #     the libdefs for this specific Flow version)
+  | |
+  | └ test_yargs.js         # <-- Tests in this directory apply to libdefs for
+  |                         #     all versions of Flow.
+  ├ color_v0.7.x/
+  ├ ...
+```
+
+Versions are semver versions with some restrictions:
```

Comment:
Does this read better? Just that sem**ver** literally is a shorthand of version so I'm reading this out loud as
> Versions are semantic version versions with some restrictions:
```suggestion
Versions are semantically versioned (semver) with some restrictions:
```